velox/connectors/hive/CMakeLists.txt

```diff
@@ -25,4 +25,5 @@ add_subdirectory(storage_adapters)
 
 if(${VELOX_BUILD_TESTING})
   add_subdirectory(tests)
+  add_subdirectory(benchmarks)
```

Comment:
Use VELOX_ENABLE_BENCHMARKS

Reply:
In `Makefile`, `VELOX_ENABLE_BENCHMARKS` can only be enabled by the `benchmarks-basic-build` target, which enables `VELOX_BUILD_MINIMAL` at the same time -> this disables `VELOX_ENABLE_HIVE_CONNECTOR`. So `make benchmarks-basic-build` will not build this benchmark as its parent directory is not built.
In this case, to build this benchmark we can only pass flags manually to cmake. Is this expected? 
I see some top level directories enables its benchmarks by `VELOX_BUILD_TESTING`, like `velox/exec`.

Reply:
@kgpai @laithsakka @pedroerp Folks, is this something you could help with?

Reply:
CC: @majetideepak